aws-sdk-java/pom.xml

```diff
@@ -15,6 +15,19 @@
         Amazon AutoScaling, etc).
     </description>
     <url>https://aws.amazon.com/sdkforjava</url>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>software.amazon.awssdk</groupId>
+                <artifactId>bom-internal</artifactId>
```

Comment:
Why does this need the BOM?

Reply:
This is something I tried to make it build with CodeBuild on this module. After this change, this module can be built successfully but other module fails. I can remove this.

Reply:
Ah okay. Yeah let's remove it for now, this module shouldn't need it.